src/Features/CSharp/Portable/SignatureHelp/InvocationExpressionSignatureHelpProvider.cs

```diff
@@ -95,9 +95,18 @@ private static bool IsArgumentListToken(InvocationExpressionSyntax expression, S
 
```

Comment:
this would need to be done for the other SigHelpProviders.  Including VB's, as well as things like ObjectCreationSigHelpprovider, etc. etc.

Note: i could have sworn we had this code at some point... is it possible we have it, but something else is accidently clearing this out?

Reply:
Yes, I'm looking at other providers now :-)